src/clj/harja/palvelin/integraatiot/paikkatietojarjestelma/tuonnit/tieverkko.clj

```diff
@@ -175,6 +201,21 @@
 
                  :default nil)))))))))
 
+(defn- multilinestring-pisin-hyppy [geom]
```

Comment:
Tämäkin näyttää ihan toimivalta, mutta mietin, että olisi tästä voinut tehdä yksinkertaisemman niin, että laskisi vectoriin kaikkien pätkien pituuden ja palauttaisi "max" funktiolla vain niistä suurimman?

Tyyliin:
(max (mapv 
(fn [p]  
(.distance (.getEndPoint p) (.getStartPoint seuraava)) ;; tämä ei tiedä mistä tuo seuraava tulee, joten voi olla,e ttei 
                                                                                            ;; tämä onnistukaan näin
) 

(line-string-seq geom)))